flake-module.nix

```diff
@@ -69,13 +76,14 @@ in
           pkgs.runCommand "toYamlFile" { buildInputs = [ pkgs.yq-go ]; } ''
             yq -P '.' ${pkgs.writeTextFile { name = "tmp.json"; text = (builtins.toJSON attrs); }} > $out
           '';
+        disableTuiToString = if config.process-compose.disable-tui then "false" else "true";
```

Comment:
You an create an new `internal` (even `readOnly`) option called `extraCliArgs` and then build it based on `config.port` and `config.tui`. Then use it in the writeShellApplication below.